adminapi/dataset/filters.py

```diff
@@ -1,16 +1,8 @@
 # XXX Deprecated
```

Comment:
Will this whole module be removed at some later point?

Reply:
Yes.  I incorporated it to this merge request as well.